pkg/controller/che/che_controller.go

```diff
@@ -608,6 +609,17 @@ func (r *ReconcileChe) Reconcile(request reconcile.Request) (reconcile.Result, e
 			}
 		}
 	}
+
+	// create ConsoleLink
+	spec := v1.ConsoleLinkSpec{Link: v1.Link{Href: "https://" + instance.Spec.Server.CheHost, Text: "Eclipse Che"},
```

Comment:
is there a way to get protocol somehow?

Reply:
yes, but consolelink support just https (https://github.com/openshift/console-operator/blob/master/manifests/00-crd-extension-console-link.yaml#L52). I'll add `if https`...